en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -844,45 +844,6 @@ result set.
 Changing Fetching Strategies
 ============================
 
-As you may know already, ``belongsTo`` and ``hasOne`` associations are loaded
-using a ``JOIN`` in the main finder query. While this improves query and
-fetching speed and allows for creating more expressive conditions when
-retrieving data, this may be a problem when you want to apply certain clauses to
-the finder query for the association, such as ``order()`` or ``limit()``.
```

Comment:
this section can be retain

Reply:
This talks about limit() which we agreed isn't useful.  Do I remove that line?